src/navigations/KeyboardWrapper.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {ViewStyle, KeyboardAvoidingView, Platform} from 'react-native';
+import dimen from '../utils/dimen';
+
+type Props = {
+  children: any;
+  keyboardVerticalOffset?: number;
+  contentContainerStyle?: ViewStyle;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Props` type definition is missing the `contentContainerStyle` property. This should be included as it's mentioned in the PR summary but not used in the component.
```diff
type Props = {
  children: any;
  keyboardVerticalOffset?: number;
- 9: };
+ 9:   contentContainerStyle?: ViewStyle;
+ 10: };
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->